src/gui/pointcloud/qgspointcloudclassifiedrendererwidget.h

```diff
@@ -70,6 +70,9 @@ class GUI_EXPORT QgsPointCloudClassifiedRendererModel : public QAbstractItemMode
   signals:
     void categoriesChanged();
 
+    //! Informs views that categories were moved (e.g., via mCategories.move()) in the model.
+    void categoriesMoved();
```

Comment:
I think in the categorysymbolrenderer this is called rowsMoved. As far as I understand this is doing the same, right? So why don't keep the same name?

Reply:
Right, I was trying to avoid a [clazy warning](https://github.com/KDE/clazy/blob/1.11/docs/checks/README-overloaded-signal.md) on overloaded signals from `QAbstractItemModel::rowsMoved()`. But I think it makes sense to change it to `rowsMoved()` to keep consistency and live with the warning as the other 2 classes that you mentioned do. What do you think? Are you OK if I force a push to amend it?

Reply:
Well I find it very confusing when similar classes have signals doing exactly the same but are named different. So I would prefer the consequent naming over clazy. But I'm not sure how it should be handled so I ping @m-kuhn whether he knows...

Reply:
I don't know if there's an official rule. Slight tendance for following existing patterns and silencing the warning

Reply:
I did `push -f` replacing `categoriesMoved()` by `rowsMoved()`, thanks!